types/testingutils/beacon_node_versioned.go

```diff
@@ -163,18 +191,28 @@ var TestingSignedBeaconBlockV = func(ks *TestKeySet, version spec.DataVersion) s
 			Message:   vBlk.Capella,
 			Signature: signBeaconObject(vBlk.Capella, types.DomainProposer, ks),
 		}
-
+	case spec.DataVersionDeneb:
+		if vBlk.Deneb == nil {
+			panic("empty block contents")
+		}
+		if vBlk.Deneb.Block == nil {
+			panic("empty block")
+		}
+		return &deneb.SignedBeaconBlock{
```

Comment:
Well noticed. I'm fixing it now. At first, I used `version.SignedBeaconBlock` because it was already being used for the blinded proposer beacon roots, strangely enough. But the correct one should be the `SignedBlindedBeaconBlock` indeed.